### ISSUE TITLE
SA1514 (ElementDocumentationHeaderMustBePrecededByBlankLine) is not getting reported for enum members

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1322]
The following code reproduces this issue:

```csharp
public struct MemberOrderHelper
{
    private enum ModifierFlags
    {
        /// <summary>
        /// No modifiers
        /// </summary>
        None = 0,
        /// <summary>
        /// Readonly modifier
        /// </summary>
        Readonly = 1,
    }
}
```

### Expected results

SA1514 is reported for the documentation comment on `Readonly`.

### Actual results

SA1514 is not reported.

**Comment by hvanbakel** [comment]
grabbed
